### PR TITLE
naive attempt at making same changes as prosauer

### LINKED ISSUE #87: automatically generate toc
- [x] bump jb in `requirements.txt` as soon as new jb release is out: https://github.com/executablebooks/jupyter-book/releases
- [x] remove `_toc.yml` from version control
- [x] mention `jb toc` in `CONTRIBUTING.md` 
- [x] add the `jb toc` command to the circleci config

### PR DESCRIPTION
**Pull request by PhiSpel**
I don't know what did not work about @prosauer 's attempt. It does work now, though.

This closes #87 and allows to delete #95 

#107 is fine, as [_toc.yml](content/_toc.yml) in /content will be ignored.

### CODE REVIEW

**Review by joergbrech — CHANGES_REQUESTED** [review]
Looks good!  But shouldn't we delete the `_toc.yml` alltogether? 

The problem with the old PR was - if I remember correctly - that we were waiting for a feature of jb, that was not yet part of an official release. We had some problems with jb autocapitalizing all words in the filename: https://github.com/executablebooks/jupyter-book/issues/595.

**Comment by PhiSpel** [comment]
> Looks good! But shouldn't we delete the `_toc.yml` alltogether?

Yes, we should, but it will not conflict when compiling.

> 
> The problem with the old PR was - if I remember correctly - that we were waiting for a feature of jb, that was not yet part of an official release. We had some problems with jb autocapitalizing all words in the filename: [executablebooks/jupyter-book#595](https://github.com/executablebooks/jupyter-book/issues/595).

When I built it, capitalisation in ToC was perfectly fine.

**Comment by joergbrech** [comment]
> Yes, we should, but it will not conflict when compiling.

Can you explain what you mean by this?

**Comment by PhiSpel** [comment]
> > Yes, we should, but it will not conflict when compiling.
> 
> Can you explain what you mean by this?

`content/_toc.yml` is overwritten when building ToC via `jupyter-book toc content/`
So, it is obsolete and should be deleted (and not added when merging #107 ), but it will not cause conflicts.

**Comment by joergbrech** [comment]
Ah I get it. Don't worry about conflicts, we should take care of them where they come up, and the conflict with #107 will be resolved once the corresponding branch will be rebased on the new master. Removing the `_toc.yml` file is part of the task associated to this issue and not #106, and the history will be clearer. So could you please remove it?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,9 +72,13 @@ In this section, it is assumed that you are in the root directory of your clone
  ```
  Jupyter-lab (or jupyter-notebook) runs in the browser. If your browser does not start automatically, you might have to copy the url from the command line output to your favorite browser. 
  * Create new Markdown files or jupyter notebooks or edit the ones that are already there. Check the [wiki](https://github.com/joergbrech/Modellbildung-und-Simulation/wiki) for writing conventions *(in German)*.
```
Comment (joergbrech) [inline CONTRIBUTING.md:74]: One more thing: Can we mention the naming conventions for file names here? Just a short summary on how the toc is generated from the file names as described in the jupyter book documentation, or maybe even better: A link to the jupyter book documentation

Comment (joergbrech) [reply CONTRIBUTING.md:74]: Ah, I see that you mention that the toc is based on the file names a few lines further down. I suppose that is enough. Sorry for the noise!

**Review by joergbrech — APPROVED** [review]

**Comment by joergbrech** [comment]
Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (PhiSpel) [reply CONTRIBUTING.md:74]: I added the conventions to the Wiki

**Comment by PhiSpel** [comment]
The order in ToC seems to be messed up with a concept:

- The order is correct for chapters 0,1,6-11
- The order is alphabetical according to titles and inserted after 11 for chapters 2-5

I don't know, what makes this difference.

**Comment by joergbrech** [comment]
Also it's fine on `master`. So it's specific to the Continuous Integration run *and* the branch associated with #107.

**Comment by PhiSpel** [comment]
It actually does work on my local engine..

**Pull request merged**